src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -203,12 +203,179 @@ Copyright (c) .NET Foundation. All rights reserved.
     </EmbeddedResource>
   </ItemGroup>
 
+  <!--
+    Determine the version (including patch) of ASP.NET Core to target.
+    When targeting ASP.NET Core, the TargetFramework is used to specify the major and minor version of the runtime to use.
+    By default, the patch version is inferred. The general logic is that self-contained apps will target the latest patch
+    that the SDK knows about, while framework-dependent apps will target the ".0" patch (and roll forward to the latest
+    patch installed at runtime).
+
+    The TargetLatestAspNetCoreRuntimePatch property can be set to true or false to explicitly opt in or out of the logic
```

Comment:
Does it make sense to opt-in on latest only to one particular framework reference but not others? Just wondering if all should key off of the TargetLatestRuntimePatch property.

Reply:
We added the separate flag for source-build SDKs. The RedHat SDK sets TargetLatestAspNetCoreRuntimePatch=true because users in that environment run from NuGet packages, not a shared framework. I think it's worth dropping 3.0, but we need it in 2.x.